scripts/change_url

```diff
@@ -1,42 +1,104 @@
 #!/bin/bash
 
 #=================================================
-# GENERIC START
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
 #=================================================
 
-# Import common cmd
-source ./experimental_helper.sh
-source ./_common.sh
-
-# Source YunoHost helpers
+source _common.sh
 source /usr/share/yunohost/helpers
 
-# Stop script if errors
-ynh_abort_if_errors
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
 
-# Retrive arguments
-ynh_script_progression --message="Loading installation settings..."
 old_domain=$YNH_APP_OLD_DOMAIN
-path_url=$(ynh_normalize_url_path --path_url ${YNH_APP_NEW_PATH:-'/'})
```

Comment:
I still think that you need to keep the `ynh_normalize_url_path` because, on some really old instance the url could still by not standardized).

Reply:
I think we fixed this in the core at some point, like it automatically normalize the path when reading the setting if needed, something like this ... It's already been quite a long time, it should be okay to get rid of it...